<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IconModelTest</class>
 <widget class="QMainWindow" name="IconModelTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IconModelTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddElement">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Element</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeleteElement">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Element</string>
    </property>
   </widget>
   <widget class="QListView" name="listViewIconModel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxViewIconsOnly">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>210</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>View Icons Only</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuIconModelTest">
    <property name="title">
     <string>IconModelTest</string>
    </property>
   </widget>
   <addaction name="menuIconModelTest"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
